source/confirm_conditions_met.html.md.erb

```diff
@@ -0,0 +1,31 @@
+---
+title: Confirm conditions met
+weight: 50
+---
+
+# Confirm conditions met
+Used to confirm that a candidate has met the conditions of an offer.
+
+```
+POST /applications/:uuid/confirm_conditions_met
+```
+
+## Example request
+
+```
+POST /applications/aa546741-b54a-403c-a67a-7f5aec1939d1/confirm_conditions_met
+```
+
+Example request body:
+
+```json
+<%= {} %>
+```
+
+## Example response
+
+**HTTP 204**
```

Comment:
Technically, we are returning an empty JSON response here, which is not the same as [no content](https://httpstatusdogs.com/204-no-content). Let's talk later about what to return for successful updates.

Reply:
we decided to postpone this decision.